qcfractal/server.py

```diff
@@ -100,9 +100,45 @@ def __init__(
             logfile_prefix: str=None,
 
             # Queue options
-            queue_socket: 'queue_adapter'=None,
+            queue_socket: 'BaseAdapter'=None,
             max_active_services: int=20,
             heartbeat_frequency: int=300):
+        """QCFractal initialization
+
+        Parameters
+        ----------
+        name : str, optional
+            The name of the server itself, provided when users query /information
+        port : int, optional
+            The port the server will listen on.
+        loop : IOLoop, optional
+            Provide an IOLoop to use for the server
+        compress_response : bool, optional
+            Automatic compression of responses, turn on unless behind a proxy that
+            provides this capability.
+        security : Optional[str], optional
+            The security options for the server {None, "local"}. The local security
+            option uses the database to cache users.
+        allow_read : bool, optional
+            Allow unregistered to perform GET operations on Molecule/KeywordSets/KVStore/Results/Procedures
+        ssl_options : Optional[Dict[str, str]], optional
```

Comment:
The `ssl_options` are checked on `is False` down below and in the `test_managers` file, but that is not an allowed type. 

There is the block [https://github.com/MolSSI/QCFractal/blob/f3bd1ae9853362c56b3ef444c3314ad7e69a43c6/qcfractal/server.py#L146-L149](here (L146)) which appears to do nothing now, but checks for the `is False` (and can probably be safely removed), and then the `elif` [https://github.com/MolSSI/QCFractal/blob/f3bd1ae9853362c56b3ef444c3314ad7e69a43c6/qcfractal/server.py#L200-L201](here (L200)) a bit below that which does do something with `ssl_options=False`. It would be good to figure these out now since we are in here tinkering.

Reply:
Changed to `Union[bool, Dict[str, str]]`, all of the below logic is correct.